mypy/main.py

```diff
@@ -1148,6 +1148,7 @@ def read_types_packages_to_install(cache_dir: str, after_run: bool) -> List[str]
 
 def install_types(cache_dir: str,
```

Comment:
As `cache_dir` is always `options.cache_dir`, this is redundant with `options`. Therefore I would recommend either removing `cache_dir` or replacing `options` by `python_executable`.

Reply:
I implemented the second option in https://github.com/weddige/mypy/pull/1/commits.

Reply:
And the first option is implemented in https://github.com/weddige/mypy/pull/2

Reply:
I decided for the second option. If there are any objections to this, I'll be happy to revert this.